server/sonar-server/src/main/java/org/sonar/server/computation/dbcleaner/ProjectCleaner.java

```diff
@@ -59,7 +59,7 @@ public ProjectCleaner purge(DbSession session, IdUuidPair idUuidPair, Settings p
 
     PurgeConfiguration configuration = newDefaultPurgeConfiguration(projectSettings, idUuidPair);
 
-    cleanHistoricalData(session, configuration.rootProjectIdUuid().getId(), projectSettings);
+    cleanHistoricalData(session, configuration.rootProjectIdUuid().getUuid(), projectSettings);
```

Comment:
Next refactoring would be to drop the class `IdUuidPair`. Each time UUID should be used.